installing/README.md

```diff
@@ -0,0 +1,10 @@
+# Installing Nix
```

Comment:
i would have a whole chapter dedicated to nix philosophy before the installation one